### PR TITLE
[DSD-3315] updated application-default.properties

### PR DESCRIPTION
**Pull request by Rakshithb1**

### CODE REVIEW

**Review by sanchi-singh24 — APPROVED** [review]
@Rakshithb1 This we can go ahead for collab env for pre-registration module and make only 1 language as mandatory

**Review by nayakrounak — APPROVED** [review]
Approved

File: application-default.properties [hunk 1]
```diff
@@ -415,10 +415,10 @@ server.tomcat.accesslog.className=io.mosip.kernel.core.logger.config.SleuthValve
 websub.hub.url=${mosip.websub.url}/hub/
 websub.publish.url=${mosip.websub.url}/hub/
 
-mosip.mandatory-languages=eng
+mosip.mandatory-languages=
```
Comment (syedsalman3753) [inline application-default.properties:418]: Any reason why mandator-language is getting removed?

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

**Comment by sanchi-singh24** [comment]
@syedsalman3753 The changes in the language selection to be mandatory from 2 we want to bring it down to 1 for collab as this environment is for contributors so want to give them easy access to usage of pre-reg

**Review by syedsalman3753 — APPROVED** [review]

**Pull request merged**